pkg/executor/docker/executor_test.go

```diff
@@ -237,26 +244,39 @@ func (s *ExecutorTestSuite) TestDockerResourceLimitsMemory() {
 		s.T().Skip("Resource limits don't apply to containers running on Windows")
 	}
 
-	// this will give us a numerator and denominator that should end up at the
-	// same 0.1 value that 100m means
-	// https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/managing_monitoring_and_updating_the_kernel/using-cgroups-v2-to-control-distribution-of-cpu-time-for-applications_managing-monitoring-and-updating-the-kernel#proc_controlling-distribution-of-cpu-time-for-applications-by-adjusting-cpu-bandwidth_using-cgroups-v2-to-control-distribution-of-cpu-time-for-applications
```

Comment:
This is only relevant to the CPU test case above. Removed from here (it appears to have been copied)